### PR TITLE
docs: Various link and content updates

### LINKED ISSUE #2932: Variety of content updates for Gaia docs
# Problem

A variety of minor issues were found in Gaia docs during the Docusaurus migration, these are detailed below.

# Closing criteria

All issues below are fixed.

# Problem details
- Missing footer logo on main page
- Remove outdated/defunct wallets and explorers from the main section
- [/governance/proposal-types](https://msalopek.github.io/gaia/governance/proposal-types) is incorrectly named by “Parameter changes” in the Governance menu and the links for Software Upgrade IBC Client Update proposal types are missing.
- [/#cosmos-hub-wallets](https://msalopek.github.io/gaia/#cosmos-hub-wallets)
  - Keplr and Ledger have Android, iOS apps
  - “Metamask Snaps” - perhaps add a one-line explainer below
- [/getting-started/what-is-gaia](https://msalopek.github.io/gaia/getting-started/what-is-gaia): is this normal that the interchain-security and global-fee modules aren’t mentioned?
- [/hub-tutorials/gaiad#key-types](https://msalopek.github.io/gaia/hub-tutorials/gaiad#key-types): The “cosmospub” and “cosmosvalconspub” prefix are outdated, as they can’t be parsed using gaiad keys parse.
- [Migrations instructions ](https://msalopek.github.io/gaia/migration)page is a README and the instructions list isn’t sorted
- [/getting-started/installation#install-the-binaries](https://msalopek.github.io/gaia/getting-started/installation#install-the-binaries):  output are still for version v9.0.0
- [/hub-tutorials/join-testnet#version-history](https://msalopek.github.io/gaia/hub-tutorials/join-testnet#version-history) : table isn’t updated
- [/architecture/adr/adr-001-interchain-accounts](https://msalopek.github.io/gaia/architecture/adr/adr-001-interchain-accounts): the title is duplicated
- [/interchain-security](https://msalopek.github.io/gaia/interchain-security): add a link to the ICS https://interchainsecurity.dev/ page or gh repo

### PR DESCRIPTION
**Pull request by mmulji-ic**
## Description

Closes: #XXXX

Fixes a number of broken links and updates some content where services are no longer available.
Removed historic proposals which do not need to be maintained in the current set of docs.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

**Comment by mmulji-ic** [comment]
Should run this through the markdown link checker before merging to make sure that all nits have been fixed.

**Comment by mpoke** [comment]
> Should run this through the markdown link checker before merging to make sure that all nits have been fixed.

@mmulji-ic Still failing https://github.com/cosmos/gaia/actions/runs/8203120601/job/22435298975

**Comment by mmulji-ic** [comment]
> > Should run this through the markdown link checker before merging to make sure that all nits have been fixed.
> 
> @mmulji-ic Still failing https://github.com/cosmos/gaia/actions/runs/8203120601/job/22435298975

Will check again, seems definitely some existing working links are failing. Have added another [PR](https://github.com/cosmos/gaia/pull/2999) to check mdx files. The links I checked seem to be working, might need to tweak the markdown checker config some more.

**Review by MSalopek — APPROVED** [review]
LGTM

File: docs/README.md [hunk 1]
```diff
@@ -10,29 +10,29 @@ Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
 ## What is Gaia?
 
-- [Intro to the `gaia` software](./getting-started/what-is-gaia.md)
-- [Interacting with the `gaiad` binary](./hub-tutorials/gaiad.md)
+- [Intro to the `gaia` software](/getting-started/what-is-gaia.md)
```
Comment (mpoke) [inline docs/README.md]: These links no longer work. See https://github.com/cosmos/gaia/blob/mmulji-ic/fix-broken-links-in-docs-2/docs/README.md

File: docs/docs/architecture/adr/adr-002-globalfee.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ Please note that `feeCoins` does not contain zero coins. The fee coins are split
...
+- **Deterministic Minimum Fee Requirement**: For the `DeliverTx` process, it is essential to have a deterministic minimum fee requirement. In `CheckTx`, fee is checked by the `CombinedFeeRequirement(globalFees, localFees)`, which considers both `minimum-gas-prices` from `config/app.toml` and `MinimumGasPricesParam` from the globalfee Params (For more details, see [globalfee](/modules/globalfee)). `CombinedFeeRequirement` contains non-deterministic part: `minimum-gas-prices` from `app.toml`. Therefore, `CombinedFeeRequirement` cannot be used in `DeliverTx`. In `DeliverTx`, only `MinimumGasPricesParam` in globalfee Params is used for fee verification. The code implementation is shown below.
```
Comment (mpoke) [inline docs/docs/architecture/adr/adr-002-globalfee.md]: Link broken.

File: docs/docs/architecture/adr/adr-002-globalfee.md [hunk 3]
```diff
@@ -162,4 +162,4 @@ The introduction of FeeDecorator has replaced the usage of `MempoolFeeDecorator`
 
 ## References
 
-* [Documentation of the globalfee module](/modules/globalfee.md)
+* [Documentation of the globalfee module](/modules/globalfee#globalfee-module)
```
Comment (mpoke) [inline docs/docs/architecture/adr/adr-002-globalfee.md]: ditto: link broken

File: docs/docs/delegators/delegator-faq.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ order: 4
...
+People that cannot or do not want to operate [validator nodes](/validators/overview) can still participate in the staking process as delegators. Indeed, validators are not chosen based on their self-delegated stake but based on their total stake, which is the sum of their self-delegated stake and of the stake that is delegated to them. This is an important property, as it makes delegators a safeguard against validators that exhibit bad behavior. If a validator misbehaves, their delegators will move their Atoms away from them, thereby reducing their stake. Eventually, if a validator's stake falls under the top 180 addresses with highest stake, they will exit the validator set.
```
Comment (mpoke) [inline docs/docs/delegators/delegator-faq.md]: broken link

File: docs/docs/delegators/delegator-guide-cli.md
Comment (mpoke) [inline docs/docs/delegators/delegator-guide-cli.md:1]: Bunch of broken links

File: docs/docs/getting-started/README.md [hunk 5]
```diff
@@ -7,4 +7,4 @@ This folder contains tutorials related to the `gaia` application.
 
 - [What is Gaia?](./what-is-gaia.md)
 - [Installing `gaiad`](./installation.md)
-- [Joining Mainnet](./quickstart.mdx)
+- [Joining Mainnet](/getting-started/quickstart)
```
Comment (mpoke) [inline docs/docs/getting-started/README.md]: broken link

**Review by mpoke — CHANGES_REQUESTED** [review]
I haven't review the entire thing, but there are a bunch of broken links (at least on GH). Not sure if they work on the doc site.

File: docs/docs/architecture/adr/adr-002-globalfee.md (hunk 3 above)
Comment (MSalopek) [reply docs/docs/architecture/adr/adr-002-globalfee.md]: Done

File: docs/docs/delegators/delegator-faq.md (hunk 4 above)
Comment (MSalopek) [reply docs/docs/delegators/delegator-faq.md]: Oh, you're trying to navigate this on your local machine without the docs server runnning. You're just looking at the files from vscode?
The link is not broken, it just cannot navigate you to the correct resource without the docs server running.

File: docs/docs/architecture/adr/adr-002-globalfee.md [hunk 6]
```diff
@@ -121,7 +121,7 @@ Please note that `feeCoins` does not contain zero coins. The fee coins are split
...
+- **Deterministic Minimum Fee Requirement**: For the `DeliverTx` process, it is essential to have a deterministic minimum fee requirement. In `CheckTx`, fee is checked by the `CombinedFeeRequirement(globalFees, localFees)`, which considers both `minimum-gas-prices` from `config/app.toml` and `MinimumGasPricesParam` from the globalfee Params (For more details, see [globalfee](../../modules/globalfee)). `CombinedFeeRequirement` contains non-deterministic part: `minimum-gas-prices` from `app.toml`. Therefore, `CombinedFeeRequirement` cannot be used in `DeliverTx`. In `DeliverTx`, only `MinimumGasPricesParam` in globalfee Params is used for fee verification. The code implementation is shown below.
```
Comment (mpoke) [inline docs/docs/architecture/adr/adr-002-globalfee.md]: Link without the `.md` or `.mdx` extensions don't work for me in GH. 
```suggestion
- **Deterministic Minimum Fee Requirement**: For the `DeliverTx` process, it is essential to have a deterministic minimum fee requirement. In `CheckTx`, fee is checked by the `CombinedFeeRequirement(globalFees, localFees)`, which considers both `minimum-gas-prices` from `config/app.toml` and `MinimumGasPricesParam` from the globalfee Params (For more details, see [globalfee](../../modules/globalfee.md)). `CombinedFeeRequirement` contains non-deterministic part: `minimum-gas-prices` from `app.toml`. Therefore, `CombinedFeeRequirement` cannot be used in `DeliverTx`. In `DeliverTx`, only `MinimumGasPricesParam` in globalfee Params is used for fee verification. The code implementation is shown below. 
```

File: docs/docs/architecture/adr/adr-002-globalfee.md [hunk 7]
```diff
@@ -162,4 +162,4 @@ The introduction of FeeDecorator has replaced the usage of `MempoolFeeDecorator`
 
 ## References
 
-* [Documentation of the globalfee module](/modules/globalfee.md)
+* [Documentation of the globalfee module](../../modules/globalfee)
```
Comment (mpoke) [inline docs/docs/architecture/adr/adr-002-globalfee.md:170]: ditto
```suggestion
* [Documentation of the globalfee module](../../modules/globalfee.md)
```

File: docs/docs/governance/proposal-types/param-change.md [hunk 8]
```diff
@@ -1,32 +1,36 @@
 ---
-title: Parameter Change
+title: Parameter Changes
 order: 4
 ---
 
-This Cosmos Hub educational documentation aims to outline the Hub's parameters, describe their functions, and describe the potential implications of modifying each parameter. This documentation also aims to provide guidelines for creating and assessing parameter-change proposals.
+This documentation also aims to provide guidelines for creating and assessing parameter-change proposals.
```
Comment (mpoke) [inline docs/docs/governance/proposal-types/param-change.md]:
```suggestion
This documentation aims to provide guidelines for creating and assessing parameter-change proposals.
```

File: docs/docs/governance/proposal-types/param-change.md [hunk 9]
```diff
@@ -1,32 +1,36 @@
...
 
 Drafting and submitting a parameter-change governance proposal involves two kinds of risk: losing proposal deposit amounts and the potential to alter the function of the Cosmos Hub network in an undesirable way. 
 
 ## What parameters can be changed?
 
-The complete parameters of the Cosmos Hub are split up into different modules, each of which has its own set of parameters. Any of them can be updated with a Param Change Proposal. If you are technically inclined, this is the full [list of modules](https://github.com/cosmos/cosmos-sdk/tree/master/x) in the Cosmos SDK. The Cosmos Hub is built using the Cosmos SDK, but not all available modules are in use on the Hub.
+The complete parameters of the Cosmos Hub are split up into different modules, each of which has its own set of parameters. Most module's parameters can be updated by submitting a governance proposal.
```
Comment (mpoke) [inline docs/docs/governance/proposal-types/param-change.md]:
```suggestion
The complete parameters of the Cosmos Hub are split up into different modules, each of which has its own set of parameters. Most parameters can be updated by submitting a governance proposal.
```

**Review by mpoke — APPROVED** [review]
Great work

File: docs/docs/architecture/adr/adr-002-globalfee.md (hunk 6 above)
Comment (MSalopek) [reply docs/docs/architecture/adr/adr-002-globalfee.md]: Changed

File: docs/docs/architecture/adr/adr-002-globalfee.md (hunk 7 above)
Comment (MSalopek) [reply docs/docs/architecture/adr/adr-002-globalfee.md:170]: Applied

File: docs/docs/governance/proposal-types/param-change.md (hunk 8 above)
Comment (MSalopek) [reply docs/docs/governance/proposal-types/param-change.md]: Changed

File: docs/docs/governance/proposal-types/param-change.md (hunk 9 above)
Comment (MSalopek) [reply docs/docs/governance/proposal-types/param-change.md]: Changed

**Pull request merged**